test/unit-test/main.tf

```diff
@@ -75,10 +114,13 @@ module "s3_with_log_bucket" {
   providers = {
     aws.bucket-replication = aws
   }
-  bucket_prefix = "unit-test-bucket-with-logs"
-  force_destroy = true
-  log_buckets   = tomap({ "main_log_bucket" : module.dummy_s3_log_bucket.bucket })
-  tags          = local.tags
+  bucket_prefix    = "unit-test-bucket-with-logs"
+  force_destroy    = true
+  log_buckets      = tomap({ "main_log_bucket" : module.dummy_s3_log_bucket.bucket })
+  log_bucket_names = toset(["log-test-bucket-051683332738327"])
```

Comment:
Would this be better with multiple buckets to test the behaviour?

Reply:
yeah good point - will wait to resolve the above before adding another bucket :)